src/libraries/Native/Unix/System.Globalization.Native/pal_collation.c

```diff
@@ -112,11 +112,14 @@ static const UChar g_HalfFullHigherChars[] = {
 };
 static const int32_t g_HalfFullCharsLength = (sizeof(g_HalfFullHigherChars) / sizeof(UChar));
 
+// Hiragana without dakuten or handakuten for custom collation rules
```

Comment:
For non-Japanese speaker, it might be better to use voiced sound/semi-voiced sound instead of dakuten/handakuten?

Reply:
The character names in unicode is not dakuten/handakuten but voiced sound/semi-voiced sound mark. 
That's a good point. 

|code point|name|
|---|---|
|3099|COMBINING KATAKANA-HIRAGANA VOICED SOUND MARK|
|309A|COMBINING KATAKANA-HIRAGANA SEMIVOICED SOUND MARK|
|309B|KATAKANA-HIRAGANA VOICED SOUND MARK|
|309C|KATAKANA-HIRAGANA SEMI-VOICED SOUND MARK|

Reply:
I have rewritten dakuten/handakuten to [semi-]voiced sound mark.